spec/support/vcr_cassettes/vaos/v2/appointments/post_appointments_cc_200_with_provider_vpg.yml

```diff
@@ -0,0 +1,61 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://veteran.apps.va.gov/vpg/v1/patients/1012846043V576341/appointments
+    body:
+      encoding: UTF-8
+      string: '{"kind":"cc","status":"proposed","locationId":"983","serviceType":"podiatry","reasonCode":{"text":"string"},"contact":{"telecom":[{"type":"phone","value":"2762740095"},{"type":"email","value":"jacqueline.morgan@id.me"}]},"requestedPeriods":[{"start":"2023-01-17T07:00:00Z","end":"2023-01-17T18:59:00Z"}],"preferredTimesForPhoneCall":["Morning"],"preferredLanguage":"English","preferredLocation":{"city":"Palm Bay","state":"FL"},"practitioners":[{"identifier":[{"system":"http://hl7.org/fhir/sid/us-npi","value":"1174506877"}],"address":{"line":["590 MALABAR RD SE STE 5"],"city":"PALM BAY","state":"FL","postalCode":"32907-3108"}}]}'
```

Comment:
Who is jacqueline morgan? I don't see that user listed [here](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv).

Reply:
@devin-mccurdy It looks like you have all the reviews you need now and don't need my approval, but I just want to make sure this user and her ICN aren't real. I want to ensure they are mock. If they are, go ahead and merge :)